src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/StaffRefDataServiceImpl.java

```diff
@@ -777,13 +775,18 @@ public CaseWorkerProfile processExistingCaseWorkers(
             StaffProfileCreationRequest cwUiRequest, CaseWorkerProfile caseWorkerProfiles) {
 
         CaseWorkerProfile filteredProfile = null;
-
+        if (cwUiRequest.getSuspended() == null) {
+            cwUiRequest.setSuspended(caseWorkerProfiles.getSuspended());
+        }
         if (cwUiRequest.isSuspended()) {
```

Comment:
@vasthav-patta its the same scenario as request suspended flag = true and caseworker db flag is true
then also it will execute line 783 to update the IDAM and UP